samples/Net7Worker/EventHubCancellationToken.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Net7Worker
+{
+    public class EventHubCancellationToken
+    {
+        private readonly ILogger _logger;
+
+        public EventHubCancellationToken(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<EventHubCancellationToken>();
+        }
+
+        // Sample showing how to handle a cancellation token being received
+        // In this example, the function invocation status will be "Cancelled"
```

Comment:
I may be overlooking something, but are we asserting on _Cancelled_ function Invocation status anywhere?